docs/root/configuration/http/http_conn_man/vhds.rst

```diff
@@ -9,6 +9,100 @@ will call to dynamically fetch
+header (authority for HTTP2) entry. Resources should be named as follows:
+
+<route configuration name>/<host entry>
+
+Note that matching should be done from right to left since a host entry cannot
+contain slashes while a route configuration name can. 
+
+Joining/Reconnecting to a management server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When an Envoy instance forms a delta xDS connection to the xDS management
+server for the first time, it will receive a base configuration filtered down
+to a subset of routes that are likely useful to the Envoy instance. For
+example, these might be filtered down by routes that exist in the same
+application. The xDS management server will send this same collection of
+resources on reconnect.
+
+.. figure:: diagrams/delta_rds_connection.svg
+   :alt: Delta RDS connection
+
+Subscribing to Resources
+^^^^^^^^^^^^^^^^^^^^^^^^
+VHDS allows resources to be :ref:`subscribed <xds_protocol_delta_subscribe>` to
+using a :ref:`DeltaDiscoveryRequest <envoy_api_msg_DeltaDiscoveryRequest>`
+with the 
+:ref:`type_url <envoy_api_field_DeltaDiscoveryRequest.type_url>` set to 
+`type.googleapis.com/envoy.api.v2.route.VirtualHost` 
+and :ref:`resource_names_subscribe <envoy_api_field_DeltaDiscoveryRequest.resource_names_subscribe>` 
+set to a list of virtual host resource names for which it would like
+configuration. 
+
+If a route for the contents of a host/authority header cannot be resolved,
+the active stream is paused while a :ref:`DeltaDiscoveryRequest <envoy_api_msg_DeltaDiscoveryRequest>` is sent. 
+When a :ref:`DeltaDiscoveryResponse <envoy_api_msg_DeltaDiscoveryResponse>` is
+recieved where the host/authority header values exactly matches the 
+:ref:`aliases <envoy_api_field_resource.aliases>`,
+the route configuration is updated, the stream is resumed and processing of the
+filter chain continues.
+
+Each of the virtual hosts contained in the
+:ref:`DeltaDiscoveryResponse <envoy_api_msg_DeltaDiscoveryResponse>`
+resources field will be added to the route configuration maintained by Envoy.
+If Envoy's route configuration already contains a given virtual host, it will
+be overwritten but only if the updated virtual host is different from its
+current state. The management server needs to keep track of virtual hosts
+known to Envoy.
+
+If a virtual host is requested for which the management server does not know
+about, then the management server should respond with a 
+:ref:`DeltaDiscoveryResponse <envoy_api_msg_DeltaDiscoveryResponse>` in which
+the :ref:`resources <envoy_api_field_DeltaDiscoveryResponse.resources>` entry
+for that virtual host has the :ref:`name <envoy_api_field_resource.name>` and
+:ref:`aliases <envoy_api_field_resource.aliases>` set to the requested host
+entry and the resource unpopulated. This will allow Envoy to match the
+requested resource to the response.
+
+Updates to the route configuration entry to which a virtual host belongs will
```

Comment:
This paragraph should be updated based on what we agreed in the other thread:

- VHDS subscriptions (and the corresponding locally cached resources) go away only when an RDS update changes the `vhds` field.  Other RDS updates do not affect the VHDS data.
- Any given virtual host must be sent either via RDS or via VHDS; the same host cannot be handled in both places.  This means that if an RDS update adds a virtual host that was previously obtained via VHDS, the client should unsubscribe from that virtual host via VHDS.